python_ta/__init__.py

```diff
@@ -319,8 +327,8 @@ def reset_linter(config=None, file_linted=None):
                 linter.set_option(key, config[key])
 
     # Custom checker configuration.
-    if linter.config.pyta_type_check:
-        linter.load_plugin_modules(["python_ta.checkers.type_inference_checker"])
+    # if linter.config.pyta_type_check:
```

Comment:
In addition to deleting this commented code, you should also delete the comment immediately above it. Also you should delete the `pyta_type_check` option from the configuration file.